inc/lib/onboarding/assets/src/steps/import-site/index.js

```diff
@@ -564,6 +573,66 @@ const ImportSite = () => {
 		return true;
 	};
 
+	/**
+	 * 1.0 Perform Settings backup file stored.
+	 */
+	const performSettingsBackup = async () => {
+		dispatch( {
+			type: 'set',
+			importStatus: __( 'Taking settings backup.', 'astra-sites' ),
+		} );
+
+		const customizerContent = new FormData();
+		customizerContent.append(
+			'action',
+			'astra-sites-backup-settings'
+		);
+		customizerContent.append( '_ajax_nonce', astraSitesVars._ajax_nonce );
+
+		const status = await fetch( ajaxurl, {
+			method: 'post',
+			body: customizerContent,
+		} )
+			.then( ( response ) => response.text() )
+			.then( ( text ) => {
+				try {
+					const response = JSON.parse( text );
+					if ( response.success ) {
+						percentage += 2;
+						dispatch( {
+							type: 'set',
+							importPercent: percentage,
+						} );
+						return true;
+					}
+					throw response.data;
+				} catch ( error ) {
+					report(
+						__( 'Taking settings backup failed.', 'astra-sites' ),
+						'',
+						error?.message,
+						'',
+						'',
+						text
+					);
+
+					return false;
+				}
```

Comment:
We should not mix up `async await` with `Promises (then and catch)`.

```suggestion
					const response = JSON.parse( text );
					if ( response.success ) {
						percentage += 2;
						dispatch( {
							type: 'set',
							importPercent: percentage,
						} );
						return true;
					}
					throw response.data;
```

Reply:
We followed the previous structure. I think we can keep as same.

Reply:
@sadanandl - Yes. I understood. But If we use `.catch` then we don't need to use the `try...catch ` block.